homeassistant/components/velux/cover.py

```diff
@@ -69,13 +74,29 @@ def should_poll(self):
     @property
     def supported_features(self):
         """Flag supported features."""
-        return SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION | SUPPORT_STOP
+        supported_features = (
+            SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION | SUPPORT_STOP
+        )
+        if self.current_cover_tilt_position is not None:
+            supported_features |= (
+                SUPPORT_OPEN_TILT
+                | SUPPORT_CLOSE_TILT
+                | SUPPORT_SET_TILT_POSITION
+                | SUPPORT_STOP_TILT
+            )
+        return supported_features
 
     @property
     def current_cover_position(self):
         """Return the current position of the cover."""
         return 100 - self.node.position.position_percent
 
+    @property
+    def current_cover_tilt_position(self):
+        """Return the current position of the cover."""
+        if isinstance(self.node, Blind):
+            return 100 - self.node.orientation.position_percent
+
```

Comment:
Please return None explicitly in the other case.

Reply:
Addressed in https://github.com/home-assistant/core/pull/45596